.gitignore

```diff
@@ -1,6 +1,7 @@
 shared
 !shared/KEEP
 .vagrant/
+.DS_Store
 enhydris
```

Comment:
I wonder whether your `$HOME/.gitignore` file would be a better place for this. Generally `$HOME/.gitignore` is, I think, better for exclusions that have to do with one's own environment (such as which editor you are using). But this is an edge case, so if you think this is more logical to have it here, that's fine.

Reply:
Yes, you are right. I will remove this too.